conda.recipe/meta.yaml

```diff
@@ -25,6 +25,10 @@ test:
     - autover
   commands:
     - nosetests -vv --nologcapture
+    - autover --help
+    - autover --version
+    - autover report --help	   
+    - autover report autover python which
 
```

Comment:
Do we need the keyword `report`?  Can we not simply let people list the commands and libraries after `autover`?  Sure, it limits future keywords we might want to use, but it seems like those can all be `--` options...

Reply:
I mentioned above that:

> having subcommands is kind of left over from when there was going to be an autover command to run as part of the versioning workflow.

We no longer need that command, but I could maybe imagine having an `autover install` kind of a command, which performs the steps necessary to add autover to a project (https://github.com/ioam/autover/pull/37/files#diff-04c6e90faac2675aa89e2176d2eec7d8R22). Not sure. I don't have a strong opinion and would have no problem removing `report`.